### PR TITLE
[feature] get and count methods implemented in db and file storage

### PR DESCRIPTION
**Pull request by andres-martin**
[feature] get and count methods implemented in dB and file storage, tested with the main drive code, comments added, unit test pending

### CODE REVIEW

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -74,3 +74,22 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        ''' Returns the object based on the class name and its ID
+            , or None if not found
```
Comment (yawzyag) [inline models/engine/db_storage.py]: change the coma to the first line

**Comment by yawzyag** [comment]
The file is working fine in my machine

**Comment by yawzyag** [comment]
Unit test soon

**Comment by yawzyag** [comment]
Make a git  ignore, and erase some trash files.

**Comment by yawzyag** [comment]
You need to erase the pyc files too

**Comment by andres-martin** [comment]
ok, I am going to erase all pyc files

**Comment by yawzyag** [comment]
Please erase the pyc in models/engine too.

**Comment by andres-martin** [comment]
ok, I will do it ASAP

**Comment by yawzyag** [comment]
Greate, ready for Merge 🥇 !!

**Pull request merged**